templates/Start.html

```diff
@@ -62,47 +76,105 @@
-          src="https://user-images.githubusercontent.com/57671048/98131265-1e508380-1ee1-11eb-92b5-12c7677c08c0.png"
-          style="display:block">
-        <img class="mySlides" width=900px height=400px
-          src="https://user-images.githubusercontent.com/57671048/98131077-f103d580-1ee0-11eb-9dc3-905f3884ee1b.png"
-          style="display:block">
-        <img class="mySlides" width=900px height=400px
-          src="https://user-images.githubusercontent.com/57671048/98131204-10026780-1ee1-11eb-999b-0182a68ce529.png"
-          style="display:block">
-        <img class="mySlides" width=900px height=400px
-          src="https://user-images.githubusercontent.com/57671048/98197451-c8192a00-1f4c-11eb-8b9f-e752acce1127.png"
-          style="display:block">
-      </div>
-
-
-      <div class="center">
-
-        <form action="/RealTime" method="post">
-          <input type="submit" name="Real Time Image" value="Real Time Image" class="button" />
-        </form>
-
-        <form action="/ManualUpload" method="post">
-          <input type="submit" name="Manually Upload Image" value="Manually Upload Image" class="button" />
-        </form>
-
-      </div>
 
+  <!--Heading section-->
+  <div class="a" style="background-color:#FFE3DE;margin-top:0;position:relative;">
+    <div class="container"></div>
+    <div style="display:flex;flex-direction:column; align-items:center;
+  justify-content:center;
+  text-align:center;">
+    <div class="animation-area" style="height: 100vh;width:100%;display:flex;flex-direction:column; align-items:center;
+    justify-content:center;text-align:center;">
+    <div class="animation-area .container-sm" style="height: 70vh;width:70%;display:flex;flex-direction:column; align-items:center;
+    justify-content:center;text-align:center;background-color:#0A043C;margin:30px;color:white">
+    <div class="animation-area .container-sm" style="height: 80vh;width:90%;display:flex;flex-direction:column; align-items:center;
+    justify-content:center;text-align:center;background-color:#03506F;margin:30px;color:white;">
+    <div style="display: flex;flex-direction:row">
+      <p style="font-size: 70px;">Facial Emotion Investigator</p>
+      <img style="margin-left:-120px" src="../static/undraw_the_search_s0xf (1) 1.svg" alt="">
+    </div>
+   
+  </div>
+  </div>
+    </div>
+  </div>
+
+  <!--about section-->
+  <div style="display:flex;flex-direction:column; align-items:center;
+  justify-content:center;
+  text-align:center;">
+    <div class="animation-area" style="height: 100vh;width:100%;display:flex;flex-direction:column; align-items:center;
+    justify-content:center;text-align:center;">
+    <div class="animation-area .container-sm" style="height: 80vh;width:70%;display:flex;flex-direction:column; align-items:center;
+    justify-content:center;text-align:center;background-color:#0A043C;margin:30px;color:white">
+    <p style="font-size:xx-large;" >About the project</p>
```

Comment:
Change the line as follows 
`<p style="font-size:xx-large;" >Glimpse of the project !</p>`